src/workspace/src/home/index.ts

```diff
@@ -1,36 +1,41 @@
-import { App, getCurrentSync } from '@openfin/workspace-platform'
+import { App, getCurrentSync } from "@openfin/workspace-platform"
 import {
   Home,
   CLIProvider,
   CLISearchListenerRequest,
   CLITemplate,
   CLISearchListenerResponse,
   CLISearchResponse,
-  CLIDispatchedSearchResult
-} from '@openfin/workspace'
-import { BASE_URL } from '../consts'
-import { deletePage, getPage, launchPage } from '../browser'
-import { getAppsAndPages, HOME_ACTION_DELETE_PAGE } from './utils'
-import { execute } from '../services/executions'
-import { getUserResult, getUserToSwitch, switchUser } from '../user'
-import { getNlpResults } from './nlpProvider'
-
-const PROVIDER_ID = 'adaptive-home-provider'
-
-export async function registerHome(): Promise<void> {
+  CLIDispatchedSearchResult,
+  HomeRegistration,
+} from "@openfin/workspace"
+import { deletePage, getPage, launchPage } from "../browser"
+import {
+  ADAPTIVE_LOGO,
+  getAppsAndPages,
+  HOME_ACTION_DELETE_PAGE,
+} from "./utils"
+import { execute } from "../services/executions"
+import { getUserResult, getUserToSwitch, switchUser } from "../user"
+import { getNlpResults } from "./nlpProvider"
+
+const PROVIDER_ID = "adaptive-home-provider"
+
+export async function registerHome(): Promise<HomeRegistration> {
   const queryMinLength = 3
   const loadingResult = {
-    key: 'loading',
-    title: 'Searching...',
+    key: "loading",
+    title: "Searching...",
     data: {},
+    icon: ADAPTIVE_LOGO,
```

Comment:
not sure what this fixes .. ?  (we also have icon on cliProvider conf below ..)

Reply:
There's result specifically for the loading state, by default it showed an S like markets showed an M

Reply:
ah, thx